app/testOnly/views/helpers/FieldErrorHelper.scala.html

```diff
@@ -18,6 +18,8 @@
 @import play.api.data.Field
 @import testOnly.forms.validation.ErrorMessageHelper._
 
+@this()
```

Comment:
Why is this here

Reply:
@this() is how you make a template injectable. With or without arguments.